website/content/docs/concepts/security/permissions/resource-table.mdx

```diff
@@ -5,1738 +5,179 @@ description: |-
   Resource table
 ---
 
-# Resource table
+# Resource tables
 
-The following table works as a quick cheat-sheet to help you manage your
-permissions. Note that it's not exhaustive; for brevity it does _not_ show
+The following tables work as a quick cheat-sheet to help you manage your
```

Comment:
Good catch on the singular > plural!

Reply:
Can't take any credit there. Those were on your original PR 😅